### PR TITLE
Move menu to the top

### PR DESCRIPTION
**Pull request by nistala**
#451
Basic implementation of top menu.

### CODE REVIEW

File: static/css/base.scss [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+    margin: 0;
+    padding: 0;
+    width: 100%;
+
+    .nav-item {
+      display: inline-block;
+      padding: 10px;
+
+      &.active {
+        border-bottom: 2px solid #ffffff;
```
Comment (dmorina) [inline static/css/base.scss]: outline: none

File: static/templates/base/top_navigation.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<nav class="nav js-nav-state" role="navigation" flex layout="row" layout-align="right">
+
+    <ul class="nav-items">
```
Comment (dmorina) [inline static/templates/base/top_navigation.html]: use md-tabs please as they are implemented by the specs

File: static/templates/base/index.html [hunk 3]
```diff
@@ -23,13 +23,16 @@
...
         <div class="md-toolbar-tools" flex>
-            <div flex layout="row" layout-align="center">
-                <h1 ng-if="isLoggedIn"  layout-align="left"><a href="/">DAEMO</a></h1>
-                <h1 ng-if="!isLoggedIn" layout-align="left">Next Generation Crowdsourcing</h1>
-
+            <div flex="35" layout="row" layout-align="left">
+                <h1 layout-align="left"><a href="/">DAEMO</a></h1>
+            </div>
+            <div flex layout="row" layout-align="right">
+                <ng-include ng-if="isLoggedIn" src="'/static/templates/base/top_navigation.html'"></ng-include>
```
Comment (dmorina) [inline static/templates/base/index.html]: align this with the region border so that it touches the gray area

Comment (nistala) [inline static/templates/base/index.html]: we'll do this when we work on design, this is a basic implementation as in the description.

**Comment by dmorina** [comment]
lgtm, we will revisit this when we push design changes

**Comment by maniksingh92** [comment]
lgtm

**Comment by neilthemathguy** [comment]
lgtm.

**Pull request merged**